### PR TITLE
Profiles for bootable-jar and wildfly

### PR DESCRIPTION
**Pull request by jfdenise**

### CODE REVIEW

File: helm-bootable.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+image:
```
Comment (jmesnil) [inline helm-bootable.yaml]: I don't think we need that file. 

There is nothing specific to Bootable Jar in this configuration file except you named the application image differently.

I think your example has more strength if we show that the "packaging" of the application (with wildfly-maven-plugin or wildfly-jar-maven-plugin)  does not impact the application image and the way WildFly application are deployed on K8S.

File: pom.xml [hunk 2]
```diff
@@ -60,43 +60,12 @@
...
-                    </execution>
-                </executions>
-            </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
                 <configuration>
                     <systemProperties>
-                        <imageName>quay.io/${user.name}/${project.artifactId}</imageName>
+                        <imageName>quay.io/${user.name}/${project.artifactId}-${packaging}</imageName>
```
Comment (jmesnil) [inline pom.xml]: does 'mvn clean verify` work with the wildfly-maven-plugin?
The image that is built is named `quay.io/foo/power-of-n` while this `imageName` would be resolved to `quay.io/foo/power-of-n-`.

One more argument to use the same name for the application image :)

**Review by jmesnil — CHANGES_REQUESTED** [review]

File: helm-bootable.yaml (hunk 1 above)
Comment (jfdenise) [reply helm-bootable.yaml]: Ok, using same image name.

File: pom.xml (hunk 2 above)
Comment (jfdenise) [reply pom.xml]: Done.

**Review by jmesnil — APPROVED** [review]

**Comment by jmesnil** [comment]
@jfdenise thanks!

**Pull request merged**